docs/intro.asciidoc

```diff
@@ -72,7 +73,49 @@ https://unpkg.com/@elastic/apm-rum/dist/bundles/elastic-apm-rum.umd.min.js[UNPKG
 </script>
 ----
 
-NOTE: Currently our minified JavaScript bundle is about 16KB (gzipped).
+NOTE: Currently the optimized(minified + gzipped) JavaScript bundle size is about 16KiB.
+
+
+[float]
+[[using-production-build]]
+=== Using Production Build
+
+By default, RUM agent logs all the debug messages to the browser console. These logs are very useful in development. However, they make the RUM agent bundle size larger so you should make sure to use the optimized production version when deploying your application.
+
+You can find instructions for building optimized code below for different bundlers
```

Comment:
```suggestion
```
Let's wrap this sentence into the previous paragraph since it fits in well there